src/config.erl

```diff
@@ -0,0 +1,57 @@
+-module(config).
+
+-export([load_config/0,
+         save_config/1,
+         update_config/3,
+         lookup/2]).
+
+-type config() :: [{string(), string()}].
+
+-define(CONF_DEFAULT_LOC, "/equivcheckrc").
+
+-spec update_config(config(), string(), string()) -> config().
+update_config(Config, Key, Value) ->
+    lists:keystore(Key, 1, Config, {Key, Value}).
+
+-spec lookup(config(), string()) -> string() | boolean().
+lookup(Config, Key) ->
+    case lists:keyfind(Key, 1, Config) of
+        {_, Value} -> Value;
+        _Otherwise -> false
+    end.
+
+-spec create_default() -> config().
+create_default() ->
+    [].
+    
+-spec deserialize(string()) -> config().
+deserialize(ConfigStr) ->
+    Lines = string:split(string:trim(ConfigStr), "\n", all),
+    lists:map(fun(Line) ->
+                      [Key, Value] = string:split(Line, " "),
```

Comment:
The separator "magic constant" might be extracted into a macro (similarly in `serialize`).

Reply:
Good idea, that haven't occurred to me. I added it.